### PR TITLE
Structure changes for IDA deployment

### PR DESCRIPTION
**Pull request by CitMC**
Documentation content update for IDA-specific files and folders needed for IDA deployment.

### Change scope

The changes in this Pull Request include (please tick all that apply):

* [ ] Small language/grammar fixes
* [ ] Small content fixes 
* [ ] Addition of new content
* [ ] Sample code/command updates
* [ ] Platform fixes
* [x] Other

### CODE REVIEW

**Comment by CitMC** [comment]
@coldice and @gioelebonifetto  This PR includes all changes on a structural level for the correct deployment of the second cohort of the IDA. Please revise and merge after.

File: ida-customizations/ida-course/0.0-B9lab-Blockchains/index.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+parent:
+  title: "Getting Started"
+  description: NEW TO BLOCKCHAIN OR IN NEED OF A REFRESHER?
```
[the remark this answers is not in the document]
Comment (CitMC) [reply ida-customizations/ida-course/0.0-B9lab-Blockchains/index.md:4]: No! Great catch! Will change it asap. Thank you Xavier!

File: ida-customizations/ida-course/0-blockchain-basics/1-blockchain.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ Blockchain technology is rapidly evolving and is difficult to understand in its
 
 Blockchain is a solution to a particular problem: the **double-spending problem**.
 
-![Double-spending problem due to insufficient funds](/academy/0.0-B9lab-Blockchains/images/00_14_double_spending_v2-01.png)
+![Double-spending problem due to insufficient funds](/ida-customization/ida-course/0/blockchain-basics/images/00_14_double_spending_v2-01.png)
```
[the remark this answers is not in the document]
Comment (coldice) [reply ida-customizations/ida-course/0-blockchain-basics/1-blockchain.md]: Nope, links are not being changed - these links were incorrect.
The script is not touching any content

**Comment by coldice** [comment]
DO NOT MERGE
One issue remaining, on the main platform, the Tutorials section in the mainn menu (left) is not showing the right content.

**Pull request merged**